assets/src/infrastructure/ui/inputs/style.scss

```diff
@@ -55,6 +56,7 @@
 	&.ee-select,
 	&.ee-textarea {
 		@at-root {
+			select#{&},
```

Comment:
:+1: 
SASS-master :bow:

Reply:
just copied what you were doing, so you're the SASS master, not i